variable/emissions/emissions.yaml

```diff
@@ -190,39 +190,67 @@ Carbon Sequestration|Land Use|Soil Carbon Management:
                 techniques
    unit: Mt CO2/yr
 
-Emissions|BC:
+# Variables related to the CO2 emission by power plant
+Emissions|CO2|Electricity|Biomass|Traditional:
+   description: CO2 emmissions of a typical Traditionnal Biomass power plant
+                biomass from local and pre-existing
+                resources mostly waste :domestic and agricultural waste, forest residues.
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Biomass|New and imported :
+   description: CO2 emmissions of a typical new biomass power plant
+                biomass from dedicated energy crops and biomass imports.
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Coal:
+   description: CO2 emmissions of a typical coal power plant
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Gas|OCGT:
+   description: CO2 emmissions of a typical Open Cycle Gas Turbine power plant
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Lignite:
+   description: CO2 emmissions of a typical Lignite power plant
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Nuclear:
+   description: CO2 emmissions of a typical nuclear power plant
+   unit: tons/MWh
+
+Emissions|Black Carbon:
    description: Total black carbon emissions
    unit: Mt BC/yr
 
-Emissions|BC|AFOLU:
+Emissions|Black Carbon|AFOLU:
    description: Black carbon emissions from agriculture, forestry and other
                 land use (IPCC category 3)
    unit: Mt BC/yr
 
-Emissions|BC|Energy:
+Emissions|Black Carbon|Energy:
    description: Black carbon emissions from energy use on supply and demand
                 side, including fugitive emissions from fuels (IPCC category
                 1A, 1B)
    unit: Mt BC/yr
 
-Emissions|BC|Energy|Demand|Industry:
+Emissions|Black Carbon|Energy|Demand|Industry:
```

Comment:
I think for all the variables describing the emissions per sector or use, you should stick to the definition of the corresponding variables in final-energy.yaml, to ensure consistency
eg, here I think his is referring to 
Final Energy|Industry:
   description: Final energy consumed by the industrial sector, including
                feedstocks, including agriculture and fishing
   energy source: all 
   sector: industry
   energy use: all excl. losses                 
   unit: EJ/yr

Then I would advise to name the variable
Emissions|Black Carbon|Final Energy|Industry
and same idea for all others
@danielhuppmann any opinion?